README.md

```diff
@@ -64,3 +64,23 @@ Please visit the [project on GitHub](https://github.com/radsectors/sqlshim) to v
 
 [sqlshim]: https://github.com/radsectors/sqlshim
 [sqlsrv]: https://github.com/Azure/msphpsql "Microsoft SQL Server Driver for PHP"
+
+#DBLIB Based Version (dexterfichuk)
+## Intro
+I've been using this for awhile and performed some changes to the original one. The original copy is indeed great the way it is for some, but this one has some significant changes. Below are a few of the fundamental changes, but I've done other various tweaks to get it up and working for myself.
+
+## Changes
+- Changed from the ODBC PDO driver to use DBLIB, couldn't connect with ODBC
+- Rewrote the parameter function, as it wouldn't pass in any parameters for me.
+- Had to rewrite a converter for datatypes. With DBLIB connections everything is returned as a string, so this will fix numbers and bool's into the proper format. 
+- May now be dependent on a few more packages
+- Forces the use of FreeTDS 7.2
```

Comment:
I changed to FreeTDS 7.2 because I was having problems using stored procedures on anything newer, it would break the connection.